freqtrade/optimize/backtesting.py

```diff
@@ -780,6 +780,8 @@ def _enter_trade(self, pair: str, row: Tuple, direction: LongShort,
                     # interest_rate=interest_rate,
                     orders=[],
                 )
+            elif trade.nr_of_successful_entries == 0:
+                trade.open_rate = propose_rate
```

Comment:
We do assign open_rate in `close_bt_order()` in `trade_model.py` when the order closes.

Thinking about that a bit more - i think that is the right place to adjust the stoploss - we have a condition for "first order filled" there anyway ... so that'll be the more obvious place.